Content.Server/_DV/CosmicCult/CosmicCultRuleSystem.cs

```diff
@@ -0,0 +1,709 @@
+                });
+        }
+        else if (uid.Comp.CurrentProgress >= uid.Comp.TargetProgress && cult.Comp.CurrentTier == 1 && uid.Comp.CanTierUp)
+        {
+            _monument.SetCanTierUp(uid, false);
+
+            var cultistQuery = EntityQueryEnumerator<CosmicCultComponent>();
+            while (cultistQuery.MoveNext(out var cultist, out var cultistComp))
+            {
+                _antag.SendBriefing(cultist, Loc.GetString("cosmiccult-monument-stage2-briefing", ("time", _config.GetCVar(DCCVars.CosmicCultT2RevealDelaySeconds))), Color.FromHex("#4cabb3"), _monumentAlert);
+            }
+
+            _monument.MonumentTier2(uid);
+            cult.Comp.CurrentTier = 2;
+            _monument.UpdateMonumentReqsForTier(uid, cult.Comp.CurrentTier);
+
+            Timer.Spawn(TimeSpan.FromSeconds(_config.GetCVar(DCCVars.CosmicCultT2RevealDelaySeconds)),
+                () =>
+                {
+                    //do spooky effects
+                    var sender = Loc.GetString("cosmiccult-announcement-sender");
+                    var mapData = _map.GetMap(_transform.GetMapId(cult.Comp.MonumentInGame.Owner.ToCoordinates()));
+                    _chatSystem.DispatchStationAnnouncement(uid, Loc.GetString("cosmiccult-announce-tier2-progress"), null, false, null, Color.FromHex("#4cabb3"));
+                    _chatSystem.DispatchStationAnnouncement(uid, Loc.GetString("cosmiccult-announce-tier2-warning"), null, false, null, Color.FromHex("#cae8e8"));
+                    _audio.PlayGlobal(_tier2Sound, Filter.Broadcast(), false, AudioParams.Default);
+
+                    for (var i = 0; i < Convert.ToInt16(cult.Comp.TotalCrew / 4); i++) // spawn # malign rifts equal to 25% of the playercount
+                    {
+                        if (TryFindRandomTile(out var _, out var _, out var _, out var coords))
+                        {
+                            Spawn("CosmicMalignRift", coords);
+                        }
+                    }
+
+                    var lights = EntityQueryEnumerator<PoweredLightComponent>();
+                    while (lights.MoveNext(out var light, out _))
+                    {
+                        if (!_rand.Prob(0.50f))
+                            continue;
+                        _ghost.DoGhostBooEvent(light);
+                    }
+
+                    _monument.SetCanTierUp(uid, true);
+                    UpdateCultData(uid); //instantly go up a tier if they manage it
+                    _ui.SetUiState(uid.Owner, MonumentKey.Key, new MonumentBuiState(uid.Comp)); //not sure if this is needed but I'll be safe
+                });
+
+        }
+
+        _monument.UpdateMonumentAppearance(uid, false);
+
+        Dirty(uid);
+        _ui.SetUiState(uid.Owner, MonumentKey.Key, new MonumentBuiState(uid.Comp));
+    }
+
+
+    #region De- & Conversion
```

Comment:
Remove extra blank line

```suggestion
    }

    #region De- & Conversion
```